index.html

```diff
@@ -0,0 +1,206 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+    <header class="header">
+    <nav class="nav-menu">
+        <ul class="nav-list1">
+            <li class="h-li">
+                <a href="#">Bwalya Nonde</a>
+            </li>
+            <li>
+                <img class="toggler" src="./images/menu.png" alt="menu-con">
+            </li>
+        </ul>
+    </nav>
+    </header> 
+    <section class="bgc">
+    <section>
+        <div>
+        <h1 class="h1">Bwalya Nonde</h1>
+        <h2 class="soft">Software Developer</h2>
+    </div>
+    <div class="p-div">
+        <p class="prg">
+            I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+               don’t hestiate to contact me.
+    
+        </p>
+    </div>
+        <hr class="hr1">  
+    </section>
+    <section>
+        <ul class="skills">
+            <li>
+                <h5 class="lang">Language</h5>
+                <ul class="tec">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </li>
+            <li>
+                <h5 class="lang">Frameworks</h5>
+                <ul class="tec">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </li>
+            <li>
+                <h5 class="lang">Skills</h5>
+                <ul class="tec">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+              </li> 
+        </ul>
+        <hr class="hr1">
+        <ul class="social-icon">
+            <li><img src="./images/GitHub.png" alt="github-icon"></li>
+            <li><img src="./images/Linkedin.png" alt="linkedin icon"></li>
+            <li><img src="./images/Angelist.png" alt="angel-icon"></li>
+            <li><img src="./images/Twitter.png" alt="Twitter-icon"></li>
+            <li><img src="./images/media.png" alt="m-icon"></li>
+        </ul> 
+        <div class="w-arrow">
+    <ul class="wk-arr">
+        <li><img src="./images/Group55.png" alt="down-arrow icon"></li>
+        <li>work</li>
+    </ul>
+        </div>
+    </section>
+
+</section>
+<main class="main-section">
+    <h2 class="r-work">MY RECENT WORK</h2>
```

Comment:
- [ ] The recent work section should be wrapped in a `section` tag. You can use the `section` tag instead of the `main` tag here.

![m1](https://user-images.githubusercontent.com/70483835/164933685-2c65881c-2502-45ce-ad96-a2edc87cf063.png)